diagnostic.js

```diff
@@ -27,28 +27,69 @@
 const mongoose = require('mongoose');
 mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
+const Movement = require('models/movement.js'); // model
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const create = (name, description, startYear, endYear) => {
+  Movement.create({
+    name: name,
+    description: description,
+    startYear:startYear,
+    endYear:endYear,
+  }).then(function(movement) {
+    console.log(place);
+  }).catch(function(error) {
+    console.error(error);
+  }).then(done);
+
+};
```

Comment:
There is no `done` function here, and the only reason we had on in the lesson was that (since our application was run from the console and ran to completion for each command) we needed to make sure that our MongoDB connection was being closed when the program finished. It's not necessary here.